frontend/sw360-portlet/src/main/resources/sw360.properties

```diff
@@ -7,33 +7,33 @@
 # without any warranty.
 #
 
-# where the frontend searches the backend
-backend.url= http://127.0.0.1:8080
+## where the frontend searches the backend
+# backend.url= http://127.0.0.1:8080
```

Comment:
`ThriftClients` has localhost as default. Please check whether this difference is important